lib/views/booking_requests.erb

```diff
@@ -0,0 +1,32 @@
+<% if current_user != nil %>
+  <% @bookings.each do |mybookings| %>
+    <% if mybookings.owner == current_user.user_name %>
+      <ul>
+        <li>
+        Booking name:<%= mybookings.name %>
+        </li>
+        <li>	
+        Space: <%= mybookings.room%>
+        </li>
+
+        <li>	
+        <% if mybookings.confirmation == nil %>	
+          Confirmation: waiting for confirmation.
+        <%else%>
+          Confirmation:<%= mybookings.confirmation %> 
+        <%end%>
```

Comment:
I don't like this way of laying out confirmation. Better would be to use `confirmed` as a boolean flag on `Booking` and then handle the confirmation message here, in the view.